dashboard/app/views/layouts/_header.html.haml

```diff
@@ -78,6 +86,13 @@
             - if params[:controller] !~ /^sessions/ and signin_button_enabled
               = render partial: 'shared/user_header_ajax'
 
+              - if show_header2
+                - hamburger_class = current_user ? '' : 'show_on_tablet'
+                %span{class: hamburger_class}
+                  &nbsp;
+                  &nbsp;
+                  = render file: Rails.root.join('..', 'shared', 'haml', 'hamburger.haml'), locals: {user_type: user_type}
```

Comment:
I believe this is typical usage in Ruby.  https://ruby-doc.org/core-2.2.0/File.html#method-c-join